src/applications/vaos/actions/expressCare.js

```diff
@@ -144,3 +160,61 @@ export function routeToPreviousAppointmentPage(router, current) {
     'previous',
   );
 }
+
+async function buildPreferencesDataAndUpdate(expressCare) {
```

Comment:
I'm not sure we have to set the preferences for EC requests, but I copied it over in case.